kdc/kerberos5.c

```diff
@@ -857,14 +918,21 @@ pa_enc_ts_validate(astgs_request_t r, const PA_DATA *pa)
     if (ret)
 	return ret;
 
+    if (pa_key->mkvno != NULL)
+	r->reply_kvno = *pa_key->mkvno;
```

Comment:
The mkvno is not the kvno.  It's the kvno of the master key.

We're mixing two different things here.

Reply:
Dropped.